cypress/testcases/virtualmachines/scheduling.spec.ts

```diff
@@ -14,25 +14,27 @@ describe('VM scheduling on Specific node', () => {
 
   it('Schedule VM on the Node which is Enable Maintenance Mode', () => {
     const hostList = Cypress.env('host');
-    const hostNames: string[] = hostList.map((host: any) => host.name);
-    const maintenanceNode = hostNames[0]
-    const filterMaintenanceNames = hostNames.filter(name => name !== maintenanceNode);
+
+    const hostNames: string[] = hostList.map((node: Node) => node.customName || node.name);
+
+    const maintenanceNodeName = hostNames[0]
```

Comment:
Even host is prefilled with a item in `cypress.env.json`.
How about add an early return error message if hostList is empty. 
Same in node-scheduling.spec.ts.

Reply:
Good idea, I just added a hosts util to do that.